template_content/{% if ide_jetbrains %}.idea{% endif %}/runConfigurations/Build_Beaker_application.xml.jinja

```diff
@@ -0,0 +1,35 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Build Beaker application" type="PythonConfigurationType" factoryName="Python">
+    <module name="{{ project_name }}" />
+    <option name="INTERPRETER_OPTIONS" value="" />
+    <option name="PARENT_ENVS" value="true" />
+    <envs>
+      <env name="PYTHONUNBUFFERED" value="1" />
+    </envs>
+    <option name="SDK_HOME" value="$PROJECT_DIR${% if _copier_conf.os == 'windows' %}\.venv\Scripts\python.exe{% else %}/.venv/bin/python{% endif %}" />
+    <option name="SDK_NAME" value="Python 3.11 ({{ project_name }})" />
```

Comment:
What happens if the user is on 3.10 or 3.12 here?

Reply:
@achidlow i double checked on SDK_NAME definition and it doesn't seem to be important to represent the actual python version, and can be used as a placeholder value if launch configs are meant to be shared. 

```
 SDK_NAME is often used as a placeholder or variable name to represent the name of the Software Development Kit (SDK) that you are using or configuring in your project.
 ```
 
 I tested the behaviour on windows vm and mac and it doesn't actually matter if this field exist at all or has any dummy values as pycharm will automatically infer the interpreter from sdk_home path. 
 
 I also added explicit docs that this idea setup is primarily optimized for Pycharm CE, and given the smoke tests, i think it should be safe to have some dummy values set in sdk_name. If user then shares this config to other user we have gitignore that will only commit runConfigurations subfolder, everything else in idea folder is ignored as per documentation on jetbrains. So i think we should be safe